Intro to Microarray Header

cansavvy opened the pull request:
## Purpose:
Closes #124 

## Strategy
The concepts listed on #124 are here, but that's not a lot of items. This is still in progress. 

## Concerns/Questions for reviewers:
- It's a bit thin. Trying to figure out what information should be added or I should go into more detail about. 
- Should we discuss limma a bit? 
- Any FAQs that should be added to the end? 

## Analysis Pull Request Check List (roughly in order):

### Content checks
* [x] All `{{BLANKS}}` have been replaced with the correct content.
* [x] [Sources are cited](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#citing-sources-in-text)
* [x] Seed is set (if applicable)

### Docker/Snakemake rendering components
* [x] Added the `.html` [link to the navigation bar](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#add-new-analyses-to-the-navbar).
* [x] Any not yet added packages needed for this analysis have been added to the Dockerfile and it successfully builds.
* [x] In the Docker container, [snakemake was run for rendering](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#how-to-re-render-the-notebooks).

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,61 @@ output:
+Microarray data is generally processed in groups of chips - this can lead to [batch effects](https://en.wikipedia.org/wiki/Batch_effect#:~:text=In%20molecular%20biology%2C%20a%20batch,of%20interest%20in%20an%20experiment). 
+To minimize this, all refine.bio microarray data downloads come [quantile-normalized](https://en.wikipedia.org/wiki/Quantile_normalization). 
+See the refine.bio docs for more about the microarray process steps including the [quantile normalization](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization).
+
```
I've debated about adding a `limma` package section here but we don't use it the same way we use `DESeq2` in RNA-seq. And I'm not really sure what to say about limma. It's only used in the two differential expression modules and none of the rest.

cansavvy replied:
That is fine with me. I think that makes sense.

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,61 @@ output:
+For microarray data, a `0` is not necessarily indicative of biology; your gene of interest may have not been covered by the [probes designed for the chip that was used](TODO: Put link to refine.bio docs FAQ when https://github.com/AlexsLemonade/refinebio-docs/issues/137 is addressed). 
+
+If the gene you are interested in does not have an Ensembl ID according to the [version of the annotation](TODO: Put link to refine.bio docs FAQ when https://github.com/AlexsLemonade/refinebio-docs/issues/137 is addressed) we used, it will not be reported in any refine.bio download.  
+
```
I've also debated about adding a "How to find information about a chip" and my main advice would be to look at the GPL platform since you can download probe info there. But, I'm not sure that's really something our users would be needing to do?

envest replied:
Yea this feels more like a refine.bio FAQ, not a intro to microarray FAQ. Maybe split the middle by saying you can find out more information about your refine.bio experiment in the metadata associated with each download? (what % true is that? lol)

cansavvy replied:
Ah. It may not be clear from my TODO here, but  when that link is complete it shows how to find this. information about the refine.bio experiment. I'm fine with leaving off this section, I was thinking it's probably a thing that our users don't know how to do but probably don't have an interest in doing.

cansavvy commented:
@envest , thanks for taking a first look at this header section. As we kinda briefly touched on in science team meeting, my main question is "What information about the microarray examples might our users be looking for in this document that isn't here?"   I have two ideas I commented on in this PR that could be something users would want to know, but I'm not sure. 

These header sections we expect will be something a user will look at after they have been looking through an example and potentially have a more general question about microarray or perhaps they are feeling uncomfortable about microarray concepts and are seeking some basic info in the context of refine.bio data. 

So before you take at the content of this PR, I would say you should take a look at some of our more basic microarray examples, attempting to have a mindset of a refine.bio user, and see what general questions about microarray you think would naturally arise. 
Clustering might be a good one to start with, since its the first one listed and might be something a user would go to first: https://alexslemonade.github.io/refinebio-examples/02-microarray/clustering_microarray_01_heatmap.html 

(Any specific questions about an example would not apply to this particular PR, but those are also welcome, just they'd need to be filed as a change to the specific example.)

envest commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,60 @@ output:
+
+## Introduction to Microarray technology
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
```
Hm... this feels out of place somehow. Maybe this works better at the top of the page? Kind of threw me off track

cansavvy replied:
It's exactly the same on the RNA-seq page, I can play with moving it as some kind of intro line

envest commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,60 @@ output:
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
```
Before talking strengths/weakness, what about a brief mention of how microarray data is generated? That will basically give you a word bank to talk about strengths/weaknesses. A nice open source graphic would go a long way to illustrate the process of sample collection to data matrix.

envest replied:
Could even include refine.bio as the penultimate stage, and show a smiling researcher at the final stage with clean data in a hand basket lol too much too much

cansavvy replied:
I'll probably add a graphic, put a brief sentence about how the data are generated and then link out to a resource that goes into more detail. (I just assume everyone knows about how microarray works, but you know what they say about assuming...)

cansavvy replied:
I suppose if microarray gets a graphic, RNA-seq should get a graphic. 😝

envest commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,60 @@ output:
+
+### Microarray data **limitations**:  
+
+- If a transcript doesn't have a probe designed to it, you won't get it measured; can't be used for transcript discovery [@Mantione2014]. 
```
My question -- to what extent does DNA variation impact a translated transcript's ability to hybridize, and how does that compare to an alignment tool's difficulty to align transcripts that don't match the reference perfectly, i.e. which  approach is more robust to variation? my intuition is RNA-seq but could definitely be off base

cansavvy replied:
The extent to which a SNP affects microarray hybridization and its resulting fluorescence signal probably depends on a number of things like how long the probes are, where in the probe the SNP occurs, whether the SNP is homozygous, how pervasive the SNP is in the tissues and organism. And I don't know how well these things can be accounted for without also doing a carefully designed SNP array side by side with the the gene expression array. 

I see there are [some tools](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933513/) that try to detect SNPs from gene expression microarray,  but I don't get the impression that they used very widespread-ly(?) and I don't think the [tools we use in refine.bio (e.g. SCAN)](http://docs.refine.bio/en/latest/main_text.html#processing-information) address this for microarray data (could be wrong on either of these accounts). 

@jaclyn-taroni may have a more informed opinion to add about this. 

The second follow up question based on this context is whether this is something our users will want to know about and I am suspecting no? (also could be wrong).

envest replied:
I agree -- I think it's not important for the target audience of this into... just my musing 🤔

jashapiro replied:
A huge part of my postdoc... gone by the wayside with cheap sequencing.

envest commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,60 @@ output:
+
+- If a transcript doesn't have a probe designed to it, you won't get it measured; can't be used for transcript discovery [@Mantione2014]. 
+- A chip's probe designs are only as up to date as the genome annotation at the time it was designed [@Mantione2014].
+- As is true for all techniques that involve nucleotide hybiridization (RNA-seq too); microarray probes come with some biases depending on their nucleotide sequence composition (like GC bias). 
+
+To help address these probe nucleotide composition biases, refine.bio uses [SCAN](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508193/pdf/nihms401888.pdf) normalization methods prior to your downloads [@Piccolo2012].
```
Does the SCAN comment go only with the third bullet point? Maybe don't have the space between them

cansavvy replied:
It's at least a spin-off of the third bullet point for sure. But it doesn't exaclty qualify to be in our list of "limitations". I'll think about how to possibly restructure.

envest replied:
(Maybe as a parenthetical statement with the bullet point.)

cansavvy replied:
Hmm... I don't like a super long parenthetical statement.

cansavvy replied:
I'm okay with the spacing, but if there are more complaints I will change it. (We have the same [kind of structure on RNA-seq section header](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#rna-seq-data-limitationsbiases)

envest reviewed (commented):
Great! I put in a few Qs and suggestions. I think a workflow image or two would go a long way in this case. I am happy to help search for or sketch out some ideas if you think that's one way to go.

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,60 @@ output:
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```
My thought is that if people are already downloading something from refine.bio (or are interested in doing so, that they probably already have an understanding of what gene expression data represent. But, my second thought is that if they don't this kind of basic information should probably be directly on the `Getting Started` page: https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html What do you think? (Also I probably should have mentioned the getting started page in my original comment to you).

envest replied:
I see -- yea, maybe anything that could apply equally to both Intro to Microarray and Intro to RNA-seq could go on the Getting Starting?

cansavvy replied:
Yes, that was what I was trying to get at but you said it more clearly and directly. Lol.

cansavvy replied:
>What experimental designs are useful?

In regards to this question, I don't know how to begin to answer this for our users. I guess we could say describe experimental designs are probably bad, but beyond that some are useful for some questions.

cansavvy commented:
The figures I've added of microarray and RNA-seq methods are not the best, but I think they will do for now. It took quite sometime to find a figure that illustrated what we need for this context. There's a lot of bad looking figures out there.

cansavvy commented:
@envest , thanks for your review and comments, they were quite helpful! See if you are okay with how I've incorporated your review. Note that there are also a couple adds to the RNA-seq section header too. (I added a overview figure).

cansavvy commented:
`sessionInfo`-only changes have been cleared, @jashapiro

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,67 @@ output:
     toc_float: true
 ---
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
```
You will want to use the same wording in the parallel doc, but I am only commenting here.

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,67 @@ output:
+- [Analysis of microarray experiments of gene expression profiling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435252/) [@Tarca2006].
+- [Some nice summary slides about microarray data analysis from Emory](http://web1.sph.emory.edu/users/hwu30/teaching/bioc/GE1.pdf) [@intro-microarray-slides].
+
+### Why doesn't the gene I care about show up in this microarray dataset?
+
+If a gene is not detected in any of the samples in a set through our processing, it is still kept in the `Gene` column, but it will be reported as `0`.
```
Expanding a bit: this was not my understanding of how merges happen in refine.bio. I thought (could be wrong) that if a gene was not on a chip, it would not appear in the dataset at all (no zero, no geneid). If two chips are merged where one has the gene and the other does not, I thought we did an inner join: so the gene would not show up in the combined set. This is based on my reading of http://docs.refine.bio/en/latest/faq.html#why-do-i-get-a-limited-number-of-genes-back-when-i-aggregate-samples-from-different-platforms

cansavvy replied:
I've been confused on how to know what is kept and what is not in refine.bio datasets. Had similar confusions with RNA-seq. But I agree that that paragraph seems to suggest they are dropped.

cansavvy replied:
I will do some downloading and poking at some refine.bio datasets and report back

cansavvy replied:
There are probes removed from the original chip design, but some of those might be background and other control probes. I took a look at a Affymetrix Human Gene 1.1 ST Array which originally has [`33297` probes listed in the GEOplatform](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL11532) but has `21661` probes listed in the refine.bio download. 

If I remember correctly, I think outdated probes are removed too based on [BrainArray's updated CDF files](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/). That also adds to the 12k probes removed in this example? And because I've used an ST chip as an example, it will probably have more probes removed from the original CDF than other chip designs.

Different chips do have different numbers, so the number of genes reported isn't based on whatever the chips are being mapped to. But the same chips have the same number of probes reported over any datasets they are being used in (based on only a couple datasets I downloaded) so I don't think there are signal cutoffs.

cansavvy replied:
I guess to summarize, my questions from here are, 1) is it BrainArray's CDFs (and relatively nothing else) that determines what probes/ensembl genes are reported in refine.bio and 2) What of this information do we think users need to know in this context. I don't really want to go on too much of a probe design bunny trail.

cansavvy replied:
I can download BrainArray CDFs and see if this lines up.

cansavvy replied:
I found the numbers, and it looks like its pretty directly to whatever version of brainarray was used. hugenest11 has `21661` probes reported by brainarrray in version 22 (which is what refine.bio said was used for the dataset). http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/22.0.0/ensg.asp

So I'll make microarray section header reflect this but try not to go on a bunny trail.

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,67 @@ output:
+
+Microarray data is generally processed in groups of chips - this can lead to [batch effects](https://en.wikipedia.org/wiki/Batch_effect#:~:text=In%20molecular%20biology%2C%20a%20batch,of%20interest%20in%20an%20experiment). 
+To minimize this, all refine.bio microarray data downloads come [quantile-normalized](https://en.wikipedia.org/wiki/Quantile_normalization) which enables more confident comparisons of expression levels between experiments.
+See the refine.bio docs for more about the microarray process steps including the [quantile normalization](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization).
```
```suggestion
See the refine.bio docs for more about the microarray processing steps, including the [quantile normalization](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization).
```

jashapiro reviewed (commented):
This looks like a good start. I think it covers most of what people need to know to get started. There are a few places where I think there could be more detail or more clarity, and I have some questions about the data itself, which may not be exactly what I thought.

We should probably have some comment on the fact that there are different kinds of microarrays, and which ones (generally: Affymetrix, Illumina beadarray, etc.) are on refine.bio. Question from actual ignorance: Do we have any two-color arrays (i.e. Agilient, and if so, are these processed the same way? I know SCAN.UPC can handle them, but I don't know the details).

I think most of my other comments are at the appropriate place in the documents, and can be addressed there.

jaclyn-taroni commented:
> Question from actual ignorance: Do we have any two-color arrays (i.e. Agilient, and if so, are these processed the same way? I know SCAN.UPC can handle them, but I don't know the details).

The short answer is no. The slightly longer answer is there is some processor code for two-color Agilent arrays _around_, but we didn't include any of those platforms in [our supported platforms list](https://github.com/AlexsLemonade/refinebio/blob/e61307c65143940d261f4be2f992c3f095ee190a/config/supported_microarray_platforms.csv) so I would not expect any in prod.

And the reasons why are because of all the different experimental designs that were used for two-color arrays (I suspect some of which you could detect from the expression data! https://github.com/AlexsLemonade/refinebio/issues/87) and also because the platform accession code isn't enough to distinguish one-color from two-color Agilent arrays (https://github.com/AlexsLemonade/refinebio/issues/89).

cansavvy commented on references.bib:
```diff
@@ -490,6 +575,12 @@ @Manual{Wickham2020
 }
 
+@Website{intro-microarray-slides,
+  title = {http://web1.sph.emory.edu/users/hwu30/},
```
Whoopsies

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,67 @@ output:
+
+### Microarray data **strengths**:  
+
+- Less expensive than RNA-seq - can afford more replicates and get higher statistical power [@Tarca2006]. 
```
I think its safe to say "in general" its cheaper and a faster turn around time?

jashapiro replied:
maybe "often"?

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,67 @@ output:
+
+## About quantile normalization 
+
+Microarray data is generally processed in groups of chips - this can lead to [batch effects](https://en.wikipedia.org/wiki/Batch_effect#:~:text=In%20molecular%20biology%2C%20a%20batch,of%20interest%20in%20an%20experiment). 
```
Agreed. Will be more specific with the wording to reduce confusion. Also, are you making a "Lost in Space" reference?

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -6,3 +6,67 @@ output:
+
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+
+[@microarray-video]
```
<img width="325" alt="Screen Shot 2020-10-05 at 9 07 53 AM" src="https://user-images.githubusercontent.com/23458084/95083472-906b5800-06ea-11eb-872f-2d345db7fe56.png">

This is what it looks like on my end. Do we not like this? What if I put a `Image from: @microarray-video`. 

I don't want to link the resource itself right here because I don't consider it the most helpful video (it's probably too basic for this audience and also half talks about DNA microarrays).

jashapiro replied:
Oh, yeah, that is what it looks like. I thought you were trying to link to the video. Fine as is.

cansavvy commented:
Okay, @jashapiro I've tried to add more information about chips and address your comment: 

> We should probably have some comment on the fact that there are different kinds of microarrays, and which ones (generally: Affymetrix, Illumina beadarray, etc.) are on refine.bio. Question from actual ignorance: Do we have any two-color arrays (i.e. Agilient, and if so, are these processed the same way? I know SCAN.UPC can handle them, but I don't know the details).

As well as clarify/fix the information about gene annotation on microarrays. 
See how you feel about the changes and whether you feel it's adequate or if more details are needed. I admittedly don't even say much about two-color arrays, except that refine.bio doesn't support them and here's a link. 

I also don't say much about beadchips vs affy except, yes, refine.bio supports both and here's some links.

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,82 @@
+- [Analysis of microarray experiments of gene expression profiling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435252/) [@Tarca2006].
+- [Some nice summary slides about microarray data analysis from Emory](http://web1.sph.emory.edu/users/hwu30/teaching/bioc/GE1.pdf) [@intro-microarray-slides].
+
+### Why doesn't the gene I care about show up in this microarray dataset?
```
I think there are a few things that should be addressed here: 

One is the case where people have a single experiment that they have downloaded, which you have partly addressed here with the Brainarray discussion. But some arrays do not have Brainarray annotations (all plants, but also many human arrays, according to https://github.com/AlexsLemonade/refinebio/blob/dev/config/supported_microarray_platforms.csv so this does not universally apply. The simple explanation of "it wasn't on the chip" should probably be added.

The other case is where people are using refine.bio to combine across experiments that may have been done on different chips with different gene content. This results in genes not present on both chips being dropped, and this should be discussed as well. https://docs.refine.bio/en/latest/main_text.html#aggregations

cansavvy replied:
I guess I haven't been thinking about the aggregate cases since none of our examples or planned future examples use the aggregate option.

cansavvy replied:
Okay, I've tried to better address both of these points.

jashapiro reviewed (commented):
This is making good progress. I added a few more comments here and there where I thought some clarifications and expansions are needed, as well as a few minor corrections.

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,82 @@
+
+[@microarray-video]
+
+There are many different kinds of microarray platforms. 
```
>[The two most common microarray platforms][The microarrays platforms found] on refine.bio are Affymetrix GeneChips and Illumina Beadarrays.

Did you mean for this to have a link? I dropped the brackets and redundant wording here for now.

cansavvy replied:
Okay in that case I used your prompt correctly then (I think).

cansavvy commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,82 @@
+
+### Microarray data **strengths**:  
+
+- Microarray is generally less expensive than RNA-seq - you can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray often has a faster turn-around than RNA-seq [@LCSciences2014].
```
I think the costs thing definitely still applies (for now) but I can hedge it more if we must.

cansavvy replied:
Fair enough. How do we get across to people the idea that "microarray data is still relevant even with new tech around" (I couldn't find sources that say this for us). What wording makes sense that we can support?

cansavvy replied:
I've been struggling from the beginning to get the wording right for this point. I've tracked this issue: #296 and propose we move forward with this PR otherwise. Does that sound fine @jashapiro?

cansavvy commented:
Noting that #296 is tracked, can you take a look at the rest the changes here @jashapiro and see if you believe your other points have been addressed?

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,88 @@
+
+### Why doesn't the gene I care about show up in this microarray dataset?
+
+A common and simple reason you may not see your gene of interest is that the microarray chip did not originally have probes designed to it. 
```
These changes look good. I have a few wording suggestions, and I am going to suggest this be made into a bulleted list.
```suggestion
- A common and simple reason you may not see your gene of interest is that the microarray chip used in the experiment you are analyzing did not originally have probes designed to target that gene. 
```

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,88 @@
+
+A common and simple reason you may not see your gene of interest is that the microarray chip did not originally have probes designed to it. 
+
+Refine.bio also uses [Brainarray packages](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/) to annotate the microarray probe data for microarray platforms that have this available [@Dai2005]. 
+This annotation identifies which probes map to which genes according to the updated transcriptome annotation (which likely changed since the microarray’s probes were first designed). 
+This also identifies which probes may have since become obsolete (they do not bind reliably to one location according to updated genome annotations). 
+If your gene of interest was covered by the original probes of the microarray chip and the version of the Brainarray package used maintains that it is still accurate, your gene of interest will show up in the Gene column.
+
+You can find your dataset’s microarray chip and Brainarray version information on the refine.bio dataset page and [by following these instructions](TODO: Put link to refine.bio docs FAQ when https://github.com/AlexsLemonade/refinebio-docs/issues/137 is addressed).
```
```suggestion
- Refine.bio uses [Brainarray packages](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/) to annotate the microarray probe data for microarray platforms that have this available [@Dai2005]. 
This annotation identifies which probes map to which genes according to the updated transcriptome annotation (which likely changed since the microarray’s probes were first designed). 
Some probes may have since become obsolete (they do not bind reliably to one location according to updated genome annotations), which may result in the gene they targeted being removed. 
If your gene of interest was covered by the original probes of the microarray chip and the version of the Brainarray package used maintains that it is still accurate, your gene of interest will show up in the Gene column.
You can find your dataset’s microarray chip and Brainarray version information on the refine.bio dataset page and [by following these instructions](TODO: Put link to refine.bio docs FAQ when https://github.com/AlexsLemonade/refinebio-docs/issues/137 is addressed).
```

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,88 @@
+
+You can find your dataset’s microarray chip and Brainarray version information on the refine.bio dataset page and [by following these instructions](TODO: Put link to refine.bio docs FAQ when https://github.com/AlexsLemonade/refinebio-docs/issues/137 is addressed).
+
+One additional reason you may not see a gene of interest applies only if you are refine.bio's [aggregate by species](https://docs.refine.bio/en/latest/main_text.html#aggregations) option.
```
```suggestion
- One additional reason you may not see a gene of interest applies only if you are refine.bio's [aggregate by species](https://docs.refine.bio/en/latest/main_text.html#aggregations) option.
```

jashapiro commented on 02-microarray/00-intro-to-microarray.Rmd:
```diff
@@ -1,8 +1,88 @@
+- [Analysis of microarray experiments of gene expression profiling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435252/) [@Tarca2006].
+- [Some nice summary slides about microarray data analysis from Emory](http://web1.sph.emory.edu/users/hwu30/teaching/bioc/GE1.pdf) [@intro-microarray-slides].
+
+### Why doesn't the gene I care about show up in this microarray dataset?
```
It seems a bit odd to have this nested under "More resources". Do we want a separate section of Common Questions (FAQs if you must). Either way, it seems like the list of links in "More resources" should maybe come at the end?

cansavvy replied:
It was this way in RNA-seq too. I changed both and reran doctoc.

cansavvy replied:
I didn't move resources last, because I don't want them to get lost and I figure someone with one of the "Common Questions" will find them regardless?

jashapiro reviewed (approved):
Okay! I have a couple more wording/organization suggestions, but I think this should be good to merge in, with the knowledge that more edits are coming with #296. See which of my suggestions you want to include, then merge away!
